apple/REAModule.mm

```diff
@@ -270,7 +270,7 @@ - (void)sendEventWithName:(NSString *)eventName body:(id)body
 #if __has_include(<UIKit/UIAccessibility.h>)
     auto isReducedMotion = UIAccessibilityIsReduceMotionEnabled();
 #else
-    auto isReducedMotion = false;
+    auto isReducedMotion = NSWorkspace.sharedWorkspace.accessibilityDisplayShouldReduceMotion;
```

Comment:
Which syntax should we use?
```suggestion
    auto isReducedMotion = [[NSWorkspace sharedWorkspace] accessibilityDisplayShouldReduceMotion];
```

Reply:
According to the [Apple documentation](https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/ProgrammingWithObjectiveC/EncapsulatingData/EncapsulatingData.html), both syntaxes are equivalent. 

> Getting a value using somePerson.firstName is the same as using [somePerson firstName]